snapshots/ggdc/2020-10-01/ggdc_maddison.xlsx.dvc

```diff
@@ -1,5 +1,4 @@
 meta:
-  name: Maddison Project Database (Bolt and van Zanden, 2020)
```

Comment:
@pabloarosado snapshot name and description are not used anymore in the new metadata framework. Currently when you call
```
snap = Snapshot("...")
ds = create_dataset(dest_dir, tables=[...], default_metadata=snap.metadata)
```
it'll try to fill dataset metadata from snapshot's metadata [like this](https://github.com/owid/etl/blob/master/etl/steps/data/converters.py#L49)
```
# dataset title and description are filled from origin
title=snap.origin.dataset_title_owid,
description=snap.origin.dataset_description_owid,
```
(which can be overwritten by setting `dataset.title` in YAML file)

The thing is that `dataset_title_owid` might be sometimes missing. Should we try to fall back to `dataset_title_producer`? Perhaps making either of the titles should be mandatory?

Reply:
I think the easiest solution is that `dataset_title_owid` should be mandatory, and `dataset_title_producer` should be optional (as very often the data doesn't really have an official title). Neither of the descriptions should be mandatory. Do you agree?

Reply:
Yes, I'll add it to this PR. GGDC will look like this
```
meta:
  origin:
    dataset_title_owid: Maddison Project Database
    dataset_title_producer: Maddison Project Database
```
(We could DRY it, but it's probably ok. I haven't included author and year.)